cmd/util/download.go

```diff
@@ -33,7 +33,9 @@ func DownloadResultsHandler(
 	}
 
 	if len(response.Results) == 0 {
-		return fmt.Errorf("no results found")
+		// No results doesn't mean error, so we should print out a message and return nil
+		cmd.Println("no results found")
+		return nil
 	}
```

Comment:
should we give a hint to use `bacalhau logs` instead? or maybe just fallback to fetching the logs if no result was found?

Reply:
I think a warning to use logs might be best. I'll add one.